strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/repository/proxied/SimpleProxyRepositoryArtifactResolver.java

```diff
@@ -60,8 +64,9 @@ protected InputStream onSuccessfulProxyRepositoryResponse(InputStream is,
             final RepositoryPath artifactPath = layoutProvider.resolve(repository).resolve(path);
             artifactByteStreamsCopyStrategyDeterminator.determine(repository).copy(is, tempPathOs, artifactPath);
         }
+
         ResourceCloser.close(is, logger);
-        return Files.newInputStream(tempPath);
+        return new BufferedInputStream(Files.newInputStream(tempPath));
```

Comment:
yeah, I am wondering why guys from Oracle still have not provide Buffered streams out of the box :confused: